### PR TITLE
Changed Packaging

### PR DESCRIPTION
**Pull request by scohen**
Implements ideas in
https://github.com/pinterest/elixir-thrift/issues/144

### CODE REVIEW

File: lib/thrift/generator/binary/framed/client.ex [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 
   def generate(service_module, service) do
     functions = service.functions
     |> Map.values
     |> Enum.map(&generate_handler_function(service_module, &1))
 
     quote do
-      defmodule Client.Framed do
-        alias Thrift.Clients.BinaryFramed
+      defmodule Binary.Framed.Client do
```
Comment (pinterest-ebert) [inline lib/thrift/generator/binary/framed/client.ex]: Modules should have a @moduledoc tag.

**Comment by pinterest-ebert** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).
* 3 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/pinterest/elixir-thrift/pulls/148.

File: lib/thrift/generator/binary/framed/client.ex (hunk 1 above)
Comment (jparise) [inline lib/thrift/generator/binary/framed/client.ex]: `@moduledoc false`

**Comment by pinterest-ebert** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 possible new issue (including those that may have been commented here).
* 3 fixed issues! 🎉

But beware that this branch is **[3 commits behind the `pinterest:thrift_tng` branch](https://github.com/pinterest/elixir-thrift/compare/scohen:repackaging...pinterest:thrift_tng)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/pinterest/elixir-thrift/pulls/148.

**Pull request merged**